### PR TITLE
Added form section at home.html line #292

### PR DESCRIPTION
**Pull request by Ibranista**
added form section and applied style for it. Also, added button hover effects for the form.

### CODE REVIEW

File: pages/home.html [hunk 1]
```diff
@@ -301,15 +294,26 @@ <h2 class="email-me">
...
+        class="get-in-touch-form"
+      >
+        <input
+          type="text"
+          placeholder="Full name"
+          maxlength="30"
+          name="full name"
+          required
+        />
+        <input type="text" placeholder="Email address" name="email" required />
```
Comment (leonardodiasb) [inline pages/home.html:309]: - [ ] Your form is working, but if the user types some text in the email field that is not an email, the form does not submit correctly. Kindly fix the type of this input to `email` to fix this issue.

File: pages/home.html [hunk 2]
```diff
@@ -301,15 +294,26 @@ <h2 class="email-me">
...
         <textarea
           placeholder="Enter text here"
           id=""
           cols="40"
           rows="6"
           maxlength="500"
+          name="comment"
           required
         ></textarea>
         <button class="btn-get">Get in touch</button>
```
Comment (leonardodiasb) [inline pages/home.html:319]: - [ ] Please add a type to the submit button element since all inputs should have a type attribute.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Ibranista, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Nice UI design. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sajid-munawar — APPROVED** [review]
### Hi @Ibranista , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.

> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @sajid-munawar in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**